### DISCUSSION TITLE
how to set webpack.config.js when have two folder in the project

### DISCUSSION THREAD

**Discussion by hds220** [callstack/repack#177] [General]
how to set webpack.config.js when have two folder in the project? @zamotany 
![image](https://user-images.githubusercontent.com/6418201/171372896-7921b6ab-0dda-4898-a2f1-078b31c5546e.png)

**Comment by zamotany** [comment]
`webpack.config.js` should be provided for each React Native application - in the screenshot, it looks like `epp-patient-native` is a React Native application, so you should have `epp-patient-native/webpack.config.js`.

If `epp-patient-core` is a library and not a React Native application, you don't need `webpack.config.js` there.
